vignettes/Model.Rmd

```diff
@@ -10,90 +12,365 @@ vignette: >
+
+-   $r_{EL}$: from early to late larval stage
+
+-   $r_{L}$: from late larval stage to pupal stage
+
+-   $r_P$: from pupal stage to adult stage
+
+Mosquito infection and incubation rates:
+
+-   $\Lambda_M$: the force of mosquito infection (FOIM)
+
+-   $r_{EM}$: extrinsic incubation period
+
+And mortality rates:
+
+-   $\mu_E(t)$: the early larval stage death rate
+
+-   $\mu_L(t)$: the late larval stage death rate
+
+-   $\mu_P$: the pupal death rate
+
+-   $\mu_M$: the adult mosquito death rate
+
+Larval mosquitoes experience density dependent mortality due to a carrying capacity, $K(t)$, which may change seasonally with rainfall and where $\gamma$ is the effect of density-dependence on late stage larvae compared with early stage larvae as follows:
+
+$$
+\mu_E = \mu_E^0(1+\frac{E(t)+L(t)}{K(t)}) \\
+\mu_L = \mu_L^0(1+\gamma\frac{E(t)+L(t)}{K(t)})
+$$
+
+## Model References
+
+Griffin JT, Hollingsworth TD, Okell LC, Churcher TS, White M, et al. (2010) Reducing *Plasmodium falciparum* Malaria Transmission in Africa: A Model-Based Evaluation of Intervention Strategies. PLOS Medicine 7(8): e1000324. <https://doi.org/10.1371/journal.pmed.1000324>
+
+White, M.T., Griffin, J.T., Churcher, T.S. *et al.* Modelling the impact of vector control interventions on *Anopheles gambiae* population dynamics. *Parasites Vectors* **4**, 153 (2011). <https://doi.org/10.1186/1756-3305-4-153>
+
+Griffin, J., Ferguson, N. & Ghani, A. Estimates of the changing age-burden of *Plasmodium falciparum* malaria disease in sub-Saharan Africa. *Nat Commun* **5**, 3136 (2014). <https://doi.org/10.1038/ncomms4136>
+
+Griffin, J. T., Déirdre Hollingsworth, T., Reyburn, H., Drakeley, C. J., Riley, E. M., & Ghani, A. C. (2015). Gradual acquisition of immunity to severe malaria with increasing exposure. Proceedings of the Royal Society B: Biological Sciences, 282(1801). <https://doi.org/10.1098/rspb.2014.2657>
+
+Griffin, J. T., Bhatt, S., Sinka, M. E., Gething, P. W., Lynch, M., Patouillard, E., Shutes, E., Newman, R. D., Alonso, P., Cibulskis, R. E., & Ghani, A. C. (2016). Potential for reduction of burden and local elimination of malaria by reducing Plasmodium falciparum malaria transmission: A mathematical modelling study. The Lancet Infectious Diseases, 16(4), 465--472. [https://doi.org/10.1016/S1473-3099(15)00423-5](https://doi.org/10.1016/S1473-3099(15)00423-5)
+
+# Basic simulation
+
+### Simulation code
+
+The key package function is `run_simulation()` which, in its most basic form, simply requires a number of timesteps in days. Default parameter settings assume a human population of size of 100, an initial mosquito population size of 1000 with all species in equal proportions, with no treatment interventions and no seasonality. The full parameters list can be seen in the documentation for `get_parameters()`.
+
+```{r, output.lines=6}
+library(malariasimulation)
+test_sim <- run_simulation(timesteps = 100)
+```
+
+### Output
```

Comment:
Thanks for collating this - it begs the question if we want all these outputs by default. I imagine a lot of them (e.g. immunity values) wouldn't not be needed in the vast majority of cases. It would be good to select a sensible subset of outputs as defaul as fewer will make the model run faster (less rendering) and also be more efficient for storage.

Reply:
Sounds good!